external/curl/Make.defs

```diff
@@ -0,0 +1,22 @@
+###########################################################################
+#
+# Copyright 2016 Samsung Electronics All Rights Reserved.
```

Comment:
2018

Reply:
@sunghan-chang,
Will do the changes on monday.